bokehjs/src/lib/models/scales/scale.ts

```diff
@@ -41,7 +41,7 @@ export abstract class Scale<T = number> extends Transform<T, number> {
   v_compute(xs: Arrayable<T>): ScreenArray {
     const result = new ScreenArray(xs.length)
     const {s_compute} = this
-    for (let i = 0; i < xs.length; i++) {
+    for (let i = 0; i < result.length; i++) {
```

Comment:
This and the other change seem unrelated. If you want a meaningful change, then I would recommend factoring out `xs.length` as a constant, because `.length` is always recomputed.

```suggestion
    const n = xs.length
    const result = new ScreenArray(n)
    const {s_compute} = this
    for (let i = 0; i < n; i++) {
```

Reply:
@mattpap Thank you for the remark! As it turns out this improves the performance of Chrome already, making the other change obsolete.
As it turns out the Firefox engine can compile this as a constant while Chrome is unable to do so.